### PR TITLE
fix: windows  button position

### PR DESCRIPTION
**Pull request by mikelxk**
feat: zoom in desktop
:up: deps

### CODE REVIEW

File: electron_src/main.js [hunk 1]
```diff
@@ -53,7 +53,8 @@ function createWindow() {
     minWidth: 600,
     webPreferences: {
       contextIsolation: true,
-      preload: `${__dirname}/preload.js`
+      preload: `${__dirname}/preload.js`,
+      webSecurity: false
```
Comment (listen1) [inline electron_src/main.js:57]: any problem if web security equals to true? is it related to eletron upgrade?

Comment (mikelxk) [reply electron_src/main.js:57]: The cookie op will fail due to samesite if websecurity is true.

File: electron_src/main.js [hunk 2]
```diff
@@ -66,7 +67,64 @@ function createWindow() {
   if (windowState.maximized) {
     mainWindow.maximize();
   }
-  mainWindow.show();
+  mainWindow.on('ready-to-show', () => {
+    mainWindow.webContents.setZoomLevel(windowState.zoomLevel);
```
Comment (listen1) [inline electron_src/main.js:71]: exception when start electron process:

> Uncaught Exception:
> TypeError: Error processing argument at index 0, conversion failure from undefined
> at BrowserWindow

Comment (mikelxk) [reply electron_src/main.js:71]: There's already a default zoomLevel defined in store.

Comment (listen1) [reply electron_src/main.js:71]: migration problem. former windowState data has no zoomlevel field.

**Pull request merged**